modules/nw-metallb-community-cr.adoc

```diff
@@ -33,9 +33,7 @@ Specify the same namespace that the MetalLB Operator uses.
 
 |`spec.communities`
 |`string`
-|Specifies a list of IP addresses for MetalLB to assign to services.
-You can specify multiple ranges in a single pool, they will all share the same settings.
-Specify each range in CIDR notation or as starting and ending IP addresses separated with a hyphen.
+|Specifies a list of BGP community aliases to be used in BGPAdvertisements. A community alias consists of a pair of name (alias) and value (numeric BGP community). Link the BGPAdvertisement to a community alias by referring to the alias name in its `spec.communities` field.
```

Comment:
nit: that can be used
also, the value is not numeric but in the form number:number